### DISCUSSION TITLE
How to disable the default keymaps of nvimtree?

### DISCUSSION THREAD

**Discussion by kay-adamof** [LunarVim/LunarVim#3232] [Q&A] [answered]
Hi there.

I'm new to LunarVIm and also would like to set up my custom configuration.

LunarVim has `nvimtree` as core plugin though , I'd like to disable its default keymaps for my preference.

- `"LiveFilter"` is set to keymap as `"f"`

I want to disable the `f` to use my custom `f` keymap.

In  normal Nvim, it is able to disable like this setting:

```lua
require("nvim-tree").setup({
	view = {
		mappings = {
			list = {
				{ key = "f", action = "" },
			},
		},
	},
})
``` 
Meanwhile the same setting added to `config.lua` of LunarVim overwrite all predefined configuration like keymaps , diagnostic icons etc which I'm not gonna to change them. The code is:

```lua
lvim.builtin.nvimtree.setup ={
	view = {
		mappings = {
			list = {
				{ key = "f", action = "" },
			},
		},
	},
}
```

### `How to disable default keymaps without not to chage other default configuration?`

Thanks

**Comment by LostInTheLogs** [comment, marked as answer]
Like this:
```lua 
lvim.builtin.nvimtree.setup.view.mappings.list = { 
  {key = "f", action = "" },
}
```
this only overrides the table list and not setup, view and mappings
